Vagrantfile

```diff
@@ -95,9 +95,10 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", privileged: false, inline: <<-EOF
     echo "Vagrant Box provisioned!"
-    echo "Local server address is http://localhost:3000"
-    echo "Please do $ vagrant ssh and cd to /manageiq"
+    echo "Please do $ vagrant ssh, your code will be in ~/manageiq"
     echo "and do $ bin/setup to finish configuration"
-    echo "Server can be started with $ bundle exec rake evm:start"
+    echo "Server can be started with $ bin/rails evm:start"
```

Comment:
I don't think we recommend `bin/rails` over `bundle exec rake` anywhere else in our guides for other platforms.